### ISSUE TITLE
About the td_error Calculate 1-step Q-Learning targets 

### ISSUE THREAD

**Issue by ovechou** [mansicer/MAIC#8]
In q_learner.py line 112:
`
td_error = (chosen_action_qvals - targets.detach())
`
**However**, in the reference, I see, the **td_error should be** 
`
td_error = (targets.detach() - chosen_action_qvals)
`
![image](https://user-images.githubusercontent.com/29061606/227958831-d07dd5a6-ea08-464e-a277-c469d0c49247.png)

I think if you refer to the GitHub Repository [Python MARL framework](https://github.com/oxwhirl/pymarl)? Cause in this Repository, the td_error is also:
`
td_error = (chosen_action_qvals - targets.detach())
`

**Comment by mansicer** [comment]
Hi! You are right. This part of code comes from `pymarl`. I think this may not be a problem, as the loss is actually its square?

https://github.com/mansicer/MAIC/blob/84e3e2c269097ae50d2cb5385a99989116086f96/src/learners/maic_learner.py#L113

**Comment by ovechou** [comment]
你好

**Comment by ovechou** [comment]
> Hi! You are right. This part of code comes from `pymarl`. I think this may not be a problem, as the loss is actually its square?
> 
> https://github.com/mansicer/MAIC/blob/84e3e2c269097ae50d2cb5385a99989116086f96/src/learners/maic_learner.py#L113

Yes，you are right. Thanks!
